datasets/mx/deputies/crawler.py

```diff
@@ -0,0 +1,84 @@
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def crawl_item(input_dict: dict, context: Context):
+
+    entity = context.make("Person")
+    entity.id = context.make_id(input_dict["NombreCompleto"])
+
+    h.apply_name(
+        entity,
+        full=input_dict.pop("NombreCompleto"),
+        first_name=input_dict.pop("Nombre"),
+        last_name=input_dict.pop("PrimerApellido") or ""
+        + " "
+        + input_dict.pop("SegundoApellido") or "",
+    )
+
+    if input_dict["Telefono"] is not None:
+        entity.add("phone", "+52" + input_dict.pop("Telefono"))
+
+    entity.add("email", input_dict.pop("Correo"))
+    entity.add("political", input_dict.pop("Partido"))
+
+    position = h.make_position(
+        context, "Member of the Chamber of Deputies", country="mx"
```

Comment:
```suggestion
        context, "Member of the Chamber of Deputies of Mexico", country="mx"
```

Reply:
we tend to use the labels from https://www.wikidata.org/wiki/Wikidata:WikiProject_every_politician when they make sense